### PR TITLE
at.cpp: added the Compaq 286/SLT. complains about '102-system board f…

### PR DESCRIPTION
**Pull request by rfka01**
'102-system board failure'

### CODE REVIEW

File: src/mame/drivers/at.cpp [hunk 1]
```diff
@@ -1416,6 +1418,23 @@ ROM_START( comportiii )
 	ROMX_LOAD( "109737-002.bin", 0x10001, 0x8000, CRC(8463cc41) SHA1(cb9801591e4a2cd13bbcc40739c9e675ba84c079), ROM_SKIP(1) | ROM_BIOS(1) )
 ROM_END
 
+// Compaq SLT/286 - complains about "102 - System board failure"
+ROM_START( comslt286 )
+   ROM_REGION(0x20000,"bios", 0)
+   	ROM_LOAD16_BYTE( "compaq_slt286-bios-revision_ j.2-even.bin", 0x10000, 0x8000, CRC(77e894e0) SHA1(e935e62e203ec67eaab198c15a36cc0078fd35b0))
```
Comment (ajrhacker) [inline src/mame/drivers/at.cpp]: The space in the filename should probably be removed.

File: src/mame/drivers/at.cpp [hunk 2]
```diff
@@ -1416,6 +1418,23 @@ ROM_START( comportiii )
...
+	ROM_LOAD16_BYTE( "compaq_slt286-bios-revision_j.2-odd.bin", 0x10001, 0x8000, CRC(4a0febac) SHA1(7da5ac4bc50f25063a1d1e382b8cff9b297976f8))
+ROM_END
+
+// Dell System 200 - complains about "memory overlap at 400000" but seems to work otherwise
+ROM_START( dsys200 )
+	ROM_REGION( 0x20000, "bios", 0 )
+   	ROM_LOAD16_BYTE( "version_ 3.10_a12_even.bin", 0x10000, 0x8000, CRC(5aa81939) SHA1(d9029d3708c49e72f57ae2a340429c28ec39acab))
```
Comment (ajrhacker) [inline src/mame/drivers/at.cpp]: The space in the filename should probably be removed.

File: src/mame/drivers/at.cpp [hunk 3]
```diff
@@ -1477,12 +1496,14 @@ COMP( 1993, apxenp2,   ibm5170, 0,       at486,     0,     at_state,     init_at
...
 COMP( 1995, ficpio2,   ibm5170, 0,       ficpio2,   0,     at_state,     init_atpci,     "FIC", "486-PIO-2", MACHINE_NOT_WORKING )
 COMP( 1985, k286i,     ibm5170, 0,       k286i,     0,     at_state,     init_at,        "Kaypro",      "286i", MACHINE_NOT_WORKING )
 COMP( 1991, t2000sx,   ibm5170, 0,       at386sx,   0,     at_state,     init_at,        "Toshiba",     "T2000SX", MACHINE_NOT_WORKING )
 COMP( 199?, mbc28,     ibm5170, 0,       at386sx,   0,     at_state,     init_at,        "Sanyo",       "MBC-28", MACHINE_NOT_WORKING )
 COMP( 1986, pcd2,      ibm5170, 0,       ibm5170,   0,     at_state,     init_at,        "Siemens",     "PCD-2", MACHINE_NOT_WORKING )
 COMP( 1987, comportii ,ibm5170, 0,       comportii, 0,     at_state,     init_at,        "Compaq",      "Portable II", MACHINE_NOT_WORKING )
 COMP( 1987, comportiii,ibm5170, 0,       comportiii,0,     at_state,     init_at,        "Compaq",      "Portable III", MACHINE_NOT_WORKING )
+COMP( 1988, comslt286, ibm5170, 0,       atvga,     0,     at_state,     init_at,        "Compaq",      " SLT/286", MACHINE_NOT_WORKING )
```
Comment (ajrhacker) [inline src/mame/drivers/at.cpp]: The space at the start of the system name should probably be removed.

File: src/mame/drivers/at.cpp (hunk 1 above)
Comment (rfka01) [reply src/mame/drivers/at.cpp]: Thanks, I'll correct that with the next part of the PR

**Pull request merged**